stats/stats_linux.go

```diff
@@ -0,0 +1,157 @@
+package stats
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"log/slog"
+	"os"
+	"strconv"
+	"time"
+)
+
+type Stats struct {
+	numCpus string
+
+	loadAvg1Min  string
+	loadAvg5Min  string
+	loadAvg15Min string
+
+	memTotal     string
+	memFree      string
+	memAvailable string
+}
+
+// Failures here aren't critical, so we just log them.
+func New(ctx context.Context) *Stats {
+	s := new(Stats)
+
+	if err := s.getNumCpus(); err != nil {
+		s.numCpus = "(unknown)"
+		slog.Info("failed to get cpus", "err", err)
+	}
+
+	go func() {
+		ticker := time.NewTicker(time.Second)
+		defer ticker.Stop()
+		for {
+			select {
+			case <-ticker.C:
+				if err := s.getLoadAvgInfo(); err != nil {
+					s.loadAvg1Min = "(unknown)"
```

Comment:
just leave the string empty instead of a `(unknown)` special string when you don't have a value for any of these tags. then, in parser.go, don't set tags with the empty string as value.

this way, the zero value is meaningful on its own and also consistent with the behaviour on darwin.